siliconcompiler/apps/sc_issue.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2023 Silicon Compiler Authors. All Rights Reserved.
+import sys
+import os
+import siliconcompiler
+import tarfile
+import json
+import importlib
+
+def main():
+    progname = "sc-issue"
+    chip = siliconcompiler.Chip(progname)
+    switchlist = ['-cfg',
+                  '-version',
+                  '-arg_step',
+                  '-arg_index',
+                  '-tool_task_variable',
+                  '-tool_task_option',
+                  '-loglevel']
+    description = """
+    -----------------------------------------------------------
+    Restricted SC app that generates a sharable testcase from a 
+    failed flow or runs an issue generated with this program.
+    -----------------------------------------------------------
```

Comment:
Would be helpful to include simple usage examples, I had to read the code to figure out how to use it the first time

Reply:
added